scripts/lib/CIME/build.py

```diff
@@ -423,6 +423,11 @@ def _build_model_thread(config_dir, compclass, compname, caseroot, libroot, bldr
     if get_model() != "ufs":
         compile_cmd = "SMP={} {}".format(stringify_bool(smp), compile_cmd)
 
+    if (check_for_python(cmd)):
```

Comment:
You don't need to wrap if statements in parens.

Reply:
Good catch. Resolved in commit b87f893.